_schemaV2.graphql

```diff
@@ -14504,19 +14463,6 @@ type Partner implements Node {
     page: Int
     size: Int
   ): PartnerAlertsConnection
-  alertsSummaryArtistsConnection(
```

Comment:
Sweet with the removal of this connection, I think we can also remove these fields from the `AlertType`

https://github.com/artsy/metaphysics/blob/f0fe20e0c37fc2baedd0e8d63f9ac135724f9c6c/src/schema/v2/Alerts/index.ts#L392-L404